filter_plugins/openshift_master.py

```diff
@@ -9,7 +9,7 @@
 import yaml
 
 from ansible import errors
-from ansible.runner.filter_plugins.core import bool as ansible_bool
+from ansible.plugins.filter.core import bool as ansible_bool
```

Comment:
It looks like instead of using ansible_bool from ansible.plugins.filter.core, instead we can use:
```
from ansible.constants import mk_boolean
```
and update the ansible_bool references below to mk_boolean as well.